cernopendata/modules/fixtures/data/docs/cms-the-future-is-open-2017/cms-the-future-is-open-2017.md

```diff
@@ -1,83 +1,81 @@
-
-When you decide to jump into the CMS open data yourself (and I hope you do), you will be confronted with this question: ["I have installed the CERN Virtual Machine: now what?"](/docs/getting-started-with-cms-2011-open-data) However you answer this question, I am sure that you are going to learn something. And hopefully, you will teach the rest of us something, too.
+But what about more exploratory studies where the theory hasn’t yet been invented? What about engaging undergraduate students who haven’t decided if they want to pursue theoretical or experimental work? What about examining old data for signs of new physics? What about citizen-scientists who might not have world experts on [proton-proton](http://web.mit.edu/lns/research/particle.html) and [lead-lead](http://web.mit.edu/mithig/) collisions in the building next door? And what happens if I have a great new theoretical idea after the LHC has already shut down? These were the [questions that motivated me](https://indico.cern.ch/event/639314/contributions/2721635/attachments/1540724/2415986/jthaler_Fermilab2017_OpenData.pdf) to dig into the CMS Open Data, and I hope that they might motivate some of you to take a look as well. Our two publications are a proof of principle that open collider analyses are feasible and potentially impactful.
 
+Ultimately, physics is an experimental science, and the aphorism that “data makes you smarter” holds at the most foundational level. It is true that theoretical insights have played a crucial role in solidifying the principles of fundamental physics. But almost everything we know for certain about the universe has originated from centuries of keen observations and detailed measurements. Without experimental data, physical principles would be mere speculations. With experimental data, we have an opportunity to expose the deepest structures of the universe… not just by scribbling on a chalkboard but by smashing together particles at ever-increasing energies.
 
+When you decide to jump into the CMS Open Data yourself (and I hope you do), you will be confronted with this question: [“I have installed the CERN Virtual Machine: now what?”](http://opendata.cern.ch/getting-started/CMS) However you answer this question, I am sure that you are going to learn something. And hopefully, you will teach the rest of us something, too.
```

Comment:
chagnge to relative URL to getting started

Reply:
Oops. Good catch.

Reply:
Updated! And I've also replaced two other links (for `/record/5`).